src/EnergyPlus/DXCoils.cc

```diff
@@ -2957,12 +2958,22 @@ namespace DXCoils {
 				ErrorsFound = true;
 			}
 
+			//Air nodes
 			DXCoil( DXCoilNum ).AirInNode = GetOnlySingleNode( Alphas( 5 ), ErrorsFound, CurrentModuleObject, Alphas( 1 ), NodeType_Air, NodeConnectionType_Inlet, 1, ObjectIsNotParent );
 
 			DXCoil( DXCoilNum ).AirOutNode = GetOnlySingleNode( Alphas( 6 ), ErrorsFound, CurrentModuleObject, Alphas( 1 ), NodeType_Air, NodeConnectionType_Outlet, 1, ObjectIsNotParent );
 
 			TestCompSet( CurrentModuleObject, Alphas( 1 ), Alphas( 5 ), Alphas( 6 ), "Air Nodes" );
 
+			//Check if the air inlet node is OA node, to justify whether the coil is placed in zone or not
+			EnergyPlus::OutAirNodeManager::CheckAndAddAirNodeNumber( DXCoil( DXCoilNum ).AirInNode, IsOANodeListed );
+			if ( IsOANodeListed ) {
```

Comment:
@rongpengzhang This is not the right function to use.  This will check if the node is on an OA node list, but it will also add the node to the OA list, causing it to be initialized with outdoor air conditions. Use CheckOutAirNodeNumber instead.  Also, preferred style is to add "using OutAirNodeManager::CheckOutAirNodeNumber" rather than reference the function with full namespaces.